stempy/image.cpp

```diff
@@ -28,13 +28,24 @@ using namespace std;
 
 namespace stempy {
 
+// constructor
 template <typename T>
 Image<T>::Image(uint32_t w, uint32_t h)
   : width(w), height(h), data(new T[w * h], std::default_delete<T[]>())
 {
   std::fill(this->data.get(), this->data.get() + width * height, 0);
 }
 
+// copy constructpr
+template <typename T>
+Image<T>::Image(const Image& image) 
+{
+  width = image.width; 
+  height = image.height;
+  data = image.data; 
+}
```

Comment:
Why do we need a copy constructor? We were trying to rely on using move semantics.

Reply:
This looks like it does the same thing as the default copy constructor anyways. We can probably leave it out.

Reply:
Thanks, initially I was trying to do something like "for (singleSTEMImage : allSTEMImages)" and it gave me error that I thought was because of no copy constructor. But later I realized that I did not even need to have the for loop. But I forgot to delete this copy constructor.